src/components/ChatEntry.js

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
 import './ChatEntry.css';
-import PropTypes from 'prop-types';
+import PropTypes from 'prop-types'; 
+// import TimeStamp from './TimeStamp.js';
 
 const ChatEntry = (props) => {
+  // const convertedTimeStamp = TimeStamp(props);
+  const heart = props.liked ? '❤️' : '🤍';
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">{props.timeStamp}</p>
```

Comment:
The usage you're looking for is (first uncomment the Timestamp component import on line 4. Then this line:
```javascript
        <p className="entry-time"><TimeStamp time={props.timeStamp}/></p>
```